Neos.ContentRepositoryRegistry/Classes/Command/SubprocessProjectionCatchUpCommandController.php

```diff
@@ -21,6 +21,11 @@ class SubprocessProjectionCatchUpCommandController extends CommandController
      */
     protected $contentRepositoryRegistry;
 
+    /**
+     * @param string $contentRepositoryIdentifier
+     * @param string $projectionClassName
```

Comment:
There are two new commands where you need to define the projection. If I remember correctly, then (currently) you have to specify the full class name in one. When using the other command, however, you can use an alias.

Which one is this? Or did it change lately and I have missed it?